snappiershot/snapshot.py

```diff
@@ -237,7 +279,14 @@ def _remove_special_arguments(cls, caller_info: CallerInfo) -> CallerInfo:
 
 
 class SnapshotStatus(IntEnum):
-    """ Enumeration of snapshot statuses. """
+    """ Enumeration of snapshot statuses.
+
+    Unchecked -- The snapshot was discovered, but never asserted against.
```

Comment:
[nit] mixture of past and present tense here - should be consistent (probably present tense)